static/app/views/performance/content.tsx

```diff
@@ -223,8 +225,8 @@ class PerformanceContent extends Component<Props, State> {
           </PageHeader>
           <GlobalSdkUpdateAlert />
           {this.renderError()}
-          {showOnboarding ? (
-            <Onboarding organization={organization} />
+          {showOnboarding && project ? (
```

Comment:
@Neo-Zhixing what happens if `project` is falsy? Is that possible?

Reply:
`project` can't be falsy. See the `shouldShowOnboarding` function. This is mostly there to make TypeScript happy.